gensim/test/test_translation_matrix.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# encoding: utf-8
+import os
+import unittest
+import tempfile
+import numpy as np
+
+from gensim import utils
+from gensim.models import translation_matrix
+from gensim.models import KeyedVectors
+
+module_path = os.path.dirname(__file__)  # needed because sample data files are located in the same folder
+datapath = lambda fname: os.path.join(module_path, 'test_data', fname)
+
+
+def temp_save_file():
+    # temporary data will be stored to this file
+    return os.path.join(tempfile.gettempdir(), 'transmat-en-it.pkl')
+
+
+class TestTranslationMatrix(unittest.TestCase):
+    def test_translation_matrix(self):
+        train_file = datapath("OPUS_en_it_europarl_train_one2ten.txt")
```

Comment:
Please load all needed models/files only once in `setUp` method (because you re-use this several times)